### ISSUE TITLE
Using this for a custom task/dataset

### ISSUE THREAD

**Issue by balag59** [xh-liu/CC-FPSE#5]
Hi,
Thank you for your amazing work! I'm working on an image to image translation task, from sketches of faces to their corresponding photos. Would this work well on this task and type of data? If yes, how would I go about training with a custom dataset?
Thanks!

**Comment by xh-liu** [comment]
Hi,
To adapt this approach to your task, you need to write your custom dataloader in folder "data/", and change necessary hyperparams (e.g., the dimension of the input label map).
